### PR TITLE
Feature 20 - Add select for currencies

### LINKED ISSUE #20: Feature - Add select for currencies
## Checklist ✅

- [x] The issue is present in the latest release.
- [x] I have searched the [issues](https://github.com/Platzi-Master-C8/gethired-base/issues) of this repository and believe that this is not a duplicate.

## Summary 💡
We need to add new selector for currency exchange in our chart.

## Requirements 🌈
- Add select into component <TabCalculate /> and <TabComparate />

## Motivation 🔦
Our motivation is that users choose between the currencies they prefer

### PR DESCRIPTION
**Pull request by yadurani**
## Checklist

- [x] I have followed the [contributor guideline](https://github.com/Platzi-Master-C8/gethired-base/blob/main/CONTRIBUTING.md)
- [ ] Tests are included
- [ ] Documentation is changed or added
- [x] Related issue has been created
- [x] Commits messages follows [commit guideline](https://github.com/Platzi-Master-C8/gethired-base/blob/main/CONTRIBUTING.md/#Commits)

### Type of PR

- [x] Feature
- [ ] Bugfix
- [ ] Hotfix
- [ ] Chore

### Merge strategy

- [x] Squash (default)
- [ ] Merge commit (branch merges, usually used for merges to master)
- [ ] Rebase (preserve individual commits without merge commit)

### Purpose
Create a input to select currency
> Enlist the purposes or write an abastract about it, please be detailed, remember that not all the members of the team have the context of yout tasks
### Relevant tickets

> Paste here the link to the task ticket
resolves [#20]

### Related PR's

> Paste here the links to related PR's
### Screenshots

> Add screenshots that could help others do understand even more the objective of your pr
Screen Shot 2022-01-14 at 12 27 38 PM

### CODE REVIEW

File: src/constants/index.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
-export const listSenority = ['Junior', 'Mid-Junior', 'Mid', 'Mid-Senior', 'Senior'];
 export const ListEnglish = ['A1', 'A2', 'B1', 'B2', 'C1', 'C2'];
+export const ListCurrencies = ['USD', 'AED', 'AFN', 'ALL', 'AMD', 'ANG'];
```
Comment (kevfarid) [inline src/constants/index.js]: You should create a service for the list of currencies. You can't burn this data for it would change in the future

**Review by kevfarid — CHANGES_REQUESTED** [review]

File: src/components/TabCalculate/TabCalculate.jsx [hunk 2]
```diff
@@ -4,18 +4,29 @@ import PropTypes from 'prop-types';
...
 
-import { currencyName } from 'Constants';
+import { currencyName, ListCurrencies } from 'Constants';
 
-import { changesForm, clearFormMain, fetchChartData } from 'App/CalculateSalary/slice';
+import { changesForm, clearFormMain, fetchChartData, changeCurrency } from 'App/CalculateSalary/slice';
 import { selectFormMain, selectChartData } from 'App/CalculateSalary/selectors';
 
 import { disabled } from 'Helpers';
+import { selectCurrency } from '../../app/CalculateSalary/selectors';
```
Comment (johannpereze) [inline src/components/TabCalculate/TabCalculate.jsx]: Change to relative path, please

**Review by johannpereze — CHANGES_REQUESTED** [review]
I just saw one little thing to change 😆

File: src/components/TabCalculate/TabCalculate.jsx (hunk 2 above)
Comment (johannpereze) [reply src/components/TabCalculate/TabCalculate.jsx]: Even better, move it to line 16 :D

**Review by kevfarid — APPROVED** [review]
Nice <3

**Review by johannpereze — APPROVED** [review]
LGTM

**Pull request merged**